dts/bindings/gpio/nxp,lpc-gpio.yaml

```diff
@@ -1,48 +1,23 @@
-# Copyright (c) 2019-22, NXP
+# Copyright 2024 NXP
```

Comment:
isn't this supposed to be '2019-22, 2024' ?

Reply:
no, this is a new file

Reply:
git/github is confused because the new file has the name of the old file, whereas I renamed the old file